src/test/java/seedu/address/AppParametersTest.java

```diff
@@ -37,6 +37,13 @@ public void parse_invalidConfigPath_success() {
         assertEquals(expected, AppParameters.parse(parametersStub));
     }
 
+    @Test
+    public void toStringMethod() {
+        AppParameters appParameters = new AppParameters();
+        String expected = AppParameters.class.getCanonicalName() + "{configPath=" + appParameters.getConfigPath() + "}";
+        assertEquals(expected, new AppParameters().toString());
+    }
+
```

Comment:
The provided test suite is designed to test various aspects of an AppParameters class. It appears to be well-structured, with clear and concise tests. However, there are some areas where it could be improved:

1. Naming of the toStringMethod Test Method: Test method names should clearly express what behavior they are verifying. The method toStringMethod could be named more descriptively, such as testToStringProducesExpectedOutput.

2. Use of new AppParameters().toString(): The test creates a new AppParameters object and immediately calls toString(). It would be clearer to reuse the appParameters object, which was created just before, to ensure the expected and actual outputs are based on the same data.

Reply:
@SPWwj Sorry I didn't respond to this earlier. 
1. Yes, it descriptive test names are good to have. In case there is only `@Test` method corresponding to a method being tested, it's better to name it exactly as the method being tested. Giving it a longer name doesn't add more information. Only when there are multiple `@Test` methods that we need to differentiate them using more informative names.
2. Yes, I think so too.